README.md

```diff
@@ -1,2 +1,442 @@
+
+Step2:  To get the new h5p file,Open the  http://localhost/drupal/  and Click on Add content,then click on interactive content there select the content type having the same title provided in library.json.
+
+Step3: After adding the content new H5p file will get created in this 
+
+"C:\xampp\htdocs\drupal\sites\default\files\h5p\exports" directory, use this file to upload to server
+
+
+
+### Export and upload to webserver
+
+step1:  Open the WordPress Server and Click on the H5P Content.
+
+   step2: Click on add new and select upload option and select  the H5p file present in this
+   "C:\xampp\htdocs\drupal\sites\default\files\h5p\exports" directory and then click on the create Button , make sure to  remember the H5p Id .
+
+   step3: Click on the Books > Add new,   Write the Title
+   step4: In the Custom field select name in the dropdown as h5pId and give the value (from step2 you can get the value)
+   step5: Add new Language Tag as English ,Then Click on Publish
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Creating a New Gameplay Feature
+
+
+    
+    
+ For creating new gameplay features go to the ```slide.tsx```(C:\xampp\htdocs\drupal\sites\default\files\h5p\development\FeedTheMonsterH5P\src\components\Slide.tsx) 
+ 
+ there you will find the ```SlideComponent``` where you can use your components and share data using props as per the requirement.
+   For example adding  Score-Board feature to the main gameplay screen:
+   
+   Step 1 : Create a folder named Score-Board in that create both in src\components  ```ScoreBard.tsx ``` and ```score-board.css``` file.
+
+   Step 2 : Create and Export your ```ScoreBoard``` Component along with all the necessary imports of components and Css.
+
+   Step 3 : go to the ```slide.tsx``` there in  ```SlideComponent``` import your component there and utilise your component there.
+
+   For the reference the ```ScoreBoard``` is already done and is being commented.
+
+   In order to utilise the ```ScoreBoard``` just go to ```ScoreBoard.tsx``` and uncomment all the commented part.
+
+   Here's is the code snippet for ScoreBaord
+
+   ``` import React from "react";
+import "./score-board.css";
+import scoreBoardBg from "../../../assets/images/score_v01.png";
+import { getImagePath } from "../../app";
+const ScoreBoard = (props: any) => {
+  return (
+    <div
+      className="score-board"
+      style={{
+        backgroundImage: `url(${getImagePath() + "score_v01.png"})`,
+        backgroundSize: "contain",
+        backgroundRepeat: "no-repeat",
+      }}
+    >
+       <h3 style={{ textAlign: "center", fontSize: "2.1em" }}>{props.score}</h3>
+    </div>
+  );
+};
+
+
+export default ScoreBoard;
+```
+
+
+
+
+<!-- ROADMAP -->
+## Roadmap
+
+
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- CONTRIBUTING -->
+## Contributing
+
+
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+## License
+
```

Comment:
Remove this "License" section as we have a license file already so this would be redundant information to keep updated.